### ISSUE TITLE
Build error on MacOS

### ISSUE THREAD

**Issue by jeroen** [graphql/libgraphqlparser#77]
```
Jeroens-MacBook-Pro:libgraphqlparser jeroen$ make
[  4%] [FLEX][GraphQLScanner] Building scanner with flex 2.6.4
[  8%] Generating Ast.h
[ 13%] Generating Ast.cpp
[ 17%] Generating AstVisitor.h
[ 21%] Generating c/GraphQLAst.h
[ 26%] Generating c/GraphQLAst.cpp
[ 30%] Generating c/GraphQLAstForEachConcreteType.h
[ 34%] Generating JsonVisitor.h.inc
[ 39%] Generating JsonVisitor.cpp.inc
[ 43%] [BISON][graphqlparser_bison] Building parser with bison 3.2.1
parser.ypp:8.10-12: error: invalid version requirement: 3
 %require "3"
          ^^^
make[2]: *** [parser.tab.cpp] Error 1
make[1]: *** [CMakeFiles/graphqlparser.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by amrox** [comment]
I just ran into the same issue. Did you get any further?

**Comment by jeroen** [comment]
No, I ended up running the bison generation step on Ubuntu Bionic.

**Comment by amrox** [comment]
It works for me on macOS using the latest release tag `v7.0`

**Comment by Alan-Cha** [comment]
@amrox Do you mind explaining exactly what you did? I am also experiencing the same problem and I am on macOS as well.

**Comment by amrox** [comment]
@Alan-Cha I just checked out the `v7.0` tag and built normally. At least at the time the problem was only on the `master` branch.

**Comment by dvergeylen** [comment]
Hi,

I was running the same issue under *linux* 😕

I did a `git-bissect` and found latest commit compiling is 3b64cd5, first failing is 93afeda, so ≃ 15 commits ahead of tag 0.7.0 (and 4 commits behind HEAD of master).

Unfortunately I am not familiar with bison so I couldn't find what made this fail to compile.

`git-blame` tells @swolchok did the commits himself so I guess he may have another not committed build setup? :thinking:

**Comment by idkCpp** [comment]
Can confirm this issue running Arch and with bison 3.3.2-1 and flex 2.6.4-2 on the master commit (`a292268`), tag v0.7.0 works.

**Comment by ErikWittern** [comment]
@jeroen As you state in your original issue, changing [this line](https://github.com/graphql/libgraphqlparser/blob/master/parser.ypp#L8) to `%require "3.0"` does actually fix the build process (despite it producing warnings). Do you see any reason not to make this change?
